src/SFA.DAS.AANHub.Application/Admins/Queries/GetAdminMemberResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Admins.Queries
+{
+    [ExcludeFromCodeCoverage]
+    public class GetAdminMemberResult
+    {
+        public Guid MemberId { get; set; }
+        public string? Email { get; set; }
```

Comment:
Email and Name are both mandatory so why nullable here?

Reply:
Nullable flag removed